Vagrantfile

```diff
@@ -28,20 +28,28 @@ $forwarded_ports = {}
 $subnet = "172.17.8"
 $os = "ubuntu"
 $network_plugin = "flannel"
+
+if File.exist?(CONFIG)
```

Comment:
before your changes we can rewrite `etcd_instances` in own `config.rb`, now it is possible to change only `num_instances` and `kube_master_instances`. I think require config should be placed after all vars definition.